src/gatsby-theme-carbon/components/Switcher.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Switcher from 'gatsby-theme-carbon/src/components/Switcher';
+import { SwitcherLink, SwitcherDivider } from 'gatsby-theme-carbon/src/components/Switcher/Switcher';
+
+const CustomSwitcher = props => (
+  <Switcher {...props}>
+    <SwitcherLink href="https://ibm.com/design">IBM Design</SwitcherLink>
+    <SwitcherLink href="https://ibm.com/design/language">IBM Design Language</SwitcherLink>
+    <SwitcherLink href="https://ibm.com/brand">IBM Brand Center</SwitcherLink>
+
+    <SwitcherDivider>Design disciplines</SwitcherDivider>
```

Comment:
@JennySanchez I added in the dividers, the right side menu looked weird without them...